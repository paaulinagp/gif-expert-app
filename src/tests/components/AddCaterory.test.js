import React from 'react';
import { shallow } from 'enzyme';
import AddCategory from '../../components/AddCaterory';

describe('Pruebas del componente <AddCategory />', () => {
  const setCategories = () => {};
  const wrapper = shallow(<AddCategory setCategories={setCategories} />);
  
  test('debe mostrar el componente correctamente ', () => {
    expect(wrapper).toMatchSnapshot();
  });

  test('debe de cambiar la caja de texto', () => {
    const input = wrapper.find('input');
    const value = "Hola mundo";
    const event = { target: { value } };

    input.simulate('change', event );
    
    expect(wrapper.find('h1').text().trim()).toBe(value);
  });
  
});
