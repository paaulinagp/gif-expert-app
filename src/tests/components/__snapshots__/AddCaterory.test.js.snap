// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas del componente <AddCategory /> debe mostrar el componente correctamente  1`] = `
<form
  onSubmit={[Function]}
>
  <h1 />
  <input
    onChange={[Function]}
    type="text"
    value=""
  />
</form>
`;
